### PR TITLE
django project and user app set up

### PR DESCRIPTION
**Pull request by jakemarkisohn**
Django set up, created main project and user app

### CODE REVIEW

File: back-end/user_app/views.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    def post(self, request):
+        request.data["username"] = request.data["email"]
+        user = authenticate(**request.data)
+        if user:
+            token, created = Token.objects.get_or_create(user=user)
+            return Response({"user": {"email": user.email}, "token": token.key})
+        else:
+            return Response("Sorry, something went wrong!", status=HTTP_400_BAD_REQUEST)
+
+
```
Comment (fravila08) [inline back-end/user_app/views.py:30]: This codebase seems oddly familiar 🤣

File: back-end/personal_project/settings.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-(0nt_w(cqe5lovwozrt1@e3tar06mh@espp*lc_65wvq8^l17#"
```
Comment (fravila08) [inline back-end/personal_project/settings.py:23]: Please ensure to hide your secret key in a `.env` file

File: back-end/personal_project/urls.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
```
Comment (fravila08) [inline back-end/personal_project/urls.py:21]: This isn't routing to your urls.py in your user_app

File: back-end/personal_project/settings.py [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+# Application definition
+
+INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+    "user_app",
```
Comment (fravila08) [inline back-end/personal_project/settings.py:40]: looks like you are attempting to use TokenAuthentication but your projects settings isn't formatted properly to allow that

File: back-end/personal_project/settings.py [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/4.2/howto/static-files/
+
+STATIC_URL = "static/"
+
+# Default primary key field type
+# https://docs.djangoproject.com/en/4.2/ref/settings/#default-auto-field
+
```
Comment (fravila08) [inline back-end/personal_project/settings.py:124]: Your app user model isn't included in settings.py

**Review by fravila08 — CHANGES_REQUESTED** [review]
Please take a look at your project and connect it rest_frameworks and your user_app appropriately

File: back-end/user_app/views.py (hunk 1 above)
Comment (jakemarkisohn) [reply back-end/user_app/views.py:30]: Hahaha I know I didn't want to just pull this out of your code but this is the exact user functionality I was looking for so I just recreated it and didn't look back. Now theres definitely no excuse not to have a fully functional and well styled website 🤣

**Pull request closed without merging**